playbooks/apb-kubevirt.yml

```diff
@@ -0,0 +1,10 @@
+- hosts: localhost
+  connection: local
+  gather_facts: False
+  # unset http_proxy. required for running in the CI
+  environment:
+    http_proxy: ""
+  roles:
+    - role: kubevirt
+
+- import_playbook: "{{ playbook_dir }}/storage.yml"
```

Comment:
I believe the storage is being dropped off. @rthallisey am I right ?

Reply:
Yes.